.travis.yml

```diff
@@ -1,33 +1,47 @@
-language: objective-c
-osx_image: xcode8
-branches:
-  only:
-    - master
```

Comment:
What's the reason for removing the specification for `master` branch in the build setting?